cli/config/configuration-qemu.toml.in

```diff
@@ -312,3 +312,9 @@ disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 # If you are using docker, `disable_new_netns` only works with `docker run --net=none`
 # (default: false)
 #disable_new_netns = true
+
+# If enabled, the runtime will enable experimental features.
+# Experimental features are not stable enough for production usage so far,
```

Comment:
Suggested rewrite 317-318 - I am not certain this suggestion retains the original meaning so adjust as necessary: 
At this time, experimental features are not stable enough for production. These features will break compatibility, and are prepared for a big version bump.

Reply:
Thanks! I'll modify this part as you suggested.

I've moved experimental flag to seperate PR: https://github.com/kata-containers/runtime/pull/1227 , I'll do the rewrite there.

Reply:
addressed in #1227